tools/topology/m4/dai.m4

```diff
@@ -180,6 +180,7 @@ define(`DAI_ADD',
 `undefine(`SCHEDULE_DEADLINE')'
 `undefine(`SCHEDULE_PRIORITY')'
 `undefine(`SCHEDULE_CORE')'
+`undefine(`SCHEDULE_TIMER')'
```

Comment:
@tlauda @keyonjie I've responded to this patch in an earlier PR today (you should both have a notification). We can merge the driver part now and the topology part after rework if there is no dependency ?